3-UsingAPIs/Week2/homework/ex4-diceRace.js

```diff
@@ -15,18 +15,24 @@ const rollDie = require('../../helpers/pokerDiceRoller');
 
 function rollDice() {
   const dice = [1, 2, 3, 4, 5];
-  // TODO complete this function; use Promise.race() and rollDie()
+  const promise = dice.map(rollDie);
```

Comment:
Use plural for arrays:

```suggestion
  const promises = dice.map(rollDie);
```